packages/cli-server-api/src/openStackFrameMiddleware.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import type {IncomingMessage, ServerResponse} from 'http';
+
+import {launchEditor} from '@react-native-community/cli-tools';
+
+type Options = {
+  watchFolders: ReadonlyArray<string>;
```

Comment:
@szymonrybczak Relevant to your [other comment](https://github.com/react-native-community/cli/pull/2587#pullrequestreview-2581037995), `watchFolders` is staying around at this level because it *ought to* (not currently) be used to parse out substitutions from URLs sent by the debugger, e.g. `'[metro-project]/path/to/file.js'`.

Reply:
oh, thanks for explanation!